core/controllers/classroom_test.py

```diff
@@ -231,6 +231,230 @@ def test_get(self) -> None:
         }
         self.assertDictContainsSubset(expected_dict, json_response)
 
+    def test_get_skip_no_summary_or_topic_rights(self) -> None:
+
+        no_summary_topic_id = topic_fetchers.get_new_topic_id()
```

Comment:
I don't know what a "no summary topic" is. Please explain or choose a better name.